src/platforms/stm32/src/lib/gpio_driver.c

```diff
@@ -931,14 +931,14 @@ static NativeHandlerResult consume_gpio_mailbox(Context *ctx)
             ret = gpiodriver_close(ctx);
             break;
 
-        case GPIOInvalidCmd:
+        case GPIOInvalidCmd:;
```

Comment:
I think the right fix might be placing `{` after the begin of block, and at the end of it.
C doesn't allow you to declare variables inside of a switch case without braces.

Reply:
Fixed. I agree completely, I found the colon suggestion multiple times while searching for a solution, but it didn't look right to me either, even though it did stop the warning. It looks much better enclosing the block.